pages/common/gifsicle.md

```diff
@@ -1,20 +1,32 @@
 # gifsicle
 
-> Create GIFs.
+> GIF manipulator.
 > More information: <https://www.lcdf.org/gifsicle>.
 
-- Optimise a GIF:
+- Optimize a GIF as a new file:
 
-`gifsicle --batch --optimize=3 {{amin.gif}}`
+`gifsicle {{path/to/input.gif}} --optimize=3 -o {{path/to/output.gif}}`
 
-- Make a GIF animation with gifsicle:
+- Unoptimize a GIF in place:
 
-`gifsicle --delay={{10}} --loop *.gif > {{anim.gif}}`
+`gifsicle -b {{path/to/input.gif}} --unoptimize`
 
-- Extract frames from an animation:
+- Extract a frame from an animation:
 
-`gifsicle {{anim.gif}} '#0' > {{firstframe.gif}}`
+`gifsicle {{path/to/input.gif}} '#{{0}}' > {{path/to/firstframe.gif}}`
 
-- You can also edit animations by replacing, deleting, or inserting frames:
+- Make a GIF animation from selected still/animated GIFs:
 
-`gifsicle -b {{anim.gif}} --replace '#0' {{new.gif}}`
+`gifsicle {{*.gif}} --delay={{10}} --loop > {{path/to/input.gif}}`
```

Comment:
```suggestion
`gifsicle {{*.gif}} --delay={{10}} --loop > {{path/to/output.gif}}`
```

Reply:
Good catch.

Reply:
Oh, I guess I could have used the "commit suggestion" feature. Sorry, new at this.

Reply:
No problem, it doesn't matter, as long as a commit history is there so reviewed changes can be seen.